apt-requirements.txt

```diff
@@ -44,6 +44,7 @@ python3-pip
 python3-setuptools
 python3-urllib3
 python3-wheel
+shellcheck
```

Comment:
This added requirement may warrant an email / slack announcement since it looks like its run in CI now?